client/blocks/term-tree-selector/terms.jsx

```diff
@@ -5,7 +5,7 @@ import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 import { localize } from 'i18n-calypso';
-import VirtualScroll from 'react-virtualized/VirtualScroll';
+import { List } from 'react-virtualized';
```

Comment:
The reason we had imported directly from `react-virtualized/VirtualScroll` was to avoid pulling in [the modules we don't all use](https://github.com/bvaughn/react-virtualized/blob/master/source/index.js). This will not be a problem once Webpack 2 is released, which includes [tree shaking](http://www.2ality.com/2015/12/webpack-tree-shaking.html) for this purpose. Webpack 2 is currently release candidate and may be released "soon", though has been in the works for about two years now, so not getting my hopes up 😄  We did recently pull in [`babel-plugin-transform-imports`](https://www.npmjs.com/package/babel-plugin-transform-imports) as part of #5954 which, if configured, would allow us to import as you've suggested here. Otherwise we should revert back to referencing the file directly.